Nmaksimchicov/1.HTML/доп.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
```

Comment:
Delicate file

Reply:
could you be more specific pls.

Reply:
doublicate file. you have two files with additional taks